app/server/utils/blockchain_tasks_simulator.py

```diff
@@ -17,7 +17,7 @@ def deploy_contract(kwargs, args):
     return FakeCeleryAsyncResult()
 
 def call_contract_function(kwargs, args):
-    return FakeCeleryAsyncResult(result=18)
+    return FakeCeleryAsyncResult(result=int(1e19))
```

Comment:
Sorry explained this poorly in my last review.

The issue here is that `get_wallet_balance` (and similar things like `get_allowance`) needs to return a very different sized number (~1e19) to `get_token_decimals` (~18), so they can't use an identical `call_contract_function` response.

Reply:
Oh that makes a lot more sense! Fixed and ready for re-👀